### ISSUE TITLE
Support building against network-3.0.0.0

### ISSUE THREAD

**Issue by 23Skidoo** [vincenthz/hs-socks#19]
Fails with:

```
[6 of 7] Compiling Network.Socks5.Lowlevel ( Network/Socks5/Lowlevel.hs, dist/build/Network/Socks5/Lowlevel.o )

Network/Socks5/Lowlevel.hs:10:1: error:
    Could not load module ‘Network.BSD’
    It is a member of the hidden package ‘network-2.8.0.0’.
    Perhaps you need to add ‘network’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
   |
10 | import Network.BSD
   | ^^^^^^^^^^^^^^^^^^
```

`Network.BSD` now lives in its own package, `network-bsd`.

**Comment by RyanGlScott** [comment]
#20 fixes this.

**Comment by peti** [comment]
Ping?

**Comment by vincenthz** [comment]
it's now fixed with socks-0.6.0
